faker/providers/person/__init__.py

```diff
@@ -328,3 +328,31 @@ def suffix_female(self) -> str:
     def language_name(self) -> str:
         """Generate a random i18n language name (e.g. English)."""
         return self.random_element(self.language_names)
+
+    def height(self, fmt: str = "ft") -> str:
+        """
+        Generate random height for person in feet and inches
+
+        Arguments:
+        ----------
+            fmt = format of the data, by default it will be taken as ft and inch
+
+        """
+        if fmt == "cm":
+            return f"{self.random_number(digits=2)} cms"
```

Comment:
Does the trailing *s* really make sense here?

Reply:
> Does the trailing _s_ really make sense here?

No.  There is no plural for it...